minicluster/src/main/java/tachyon/master/LocalTachyonCluster.java

```diff
@@ -360,7 +385,11 @@ public void stopTFS() throws Exception {
    */
   public void stopUFS() throws Exception {
     LOG.info("stop under storage system");
-    mMaster.cleanupUnderfs();
+    if (mUfsCluster != null) {
+      mUfsCluster.cleanup();
+    }
+    // TODO(gpang): is this line necessary?
+    System.clearProperty("tachyon.underfs.address");
```

Comment:
This should be safe to remove. Since we set it in the conf now instead of as a system property.